bin/build-production-branch.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```

Comment:
This is a very simplified version of our build scripts. I cobbled it together from the GitHub actions build, and our existing one. I have not tested it yet, but it is meant to use the new versioning bits in this PR.